CHANGELOG.md

```diff
@@ -244,3 +244,7 @@ Fixes a bug in icon size for inline `<ChannelValue>` components.
 ## v0.0.1 (May 24, 2019)
 
 Initial beta release
+
+### Updated 
+
+Updated react component library to 5.3.2
```

Comment:
You only add change logs at the top. And you add descriptions to what's been updated. You can have a look at how other packages are doing this (like this changelog in [Angular component library](https://github.com/pxblue/angular-component-library/blob/dev/CHANGELOG.md))